spec/main_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require './lib/main'
+
+RSpec.describe Enumerable do
+  describe '#my_count' do
+    it 'calls a block on an object and returns the number of elements' do
+      arr = [1, 2, 3, 4, 5]
+      expect(arr.my_count).to eql(5)
+    end
+  end
+
+  describe '#my_map' do
+    it 'returns a new array as a result of calling a block once on each item in the original array' do
+      arr = [1, 2, 3]
```

Comment:
#### `REQUIRED`
Let's use `let` here instead. I can see you use `arr = [1, 2, 3]` in four different places already. You can define this `arr` using `let` helper method after the first `describe` block and use it anywhere else. See example below:

```ruby
RSpec.describe Enumerable do
let(:arr){[1, 2, 3]}

```

Learn more about [`let` here](https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let)